src/components/privacy/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+export const Privacy = () => {
+  return (
+    <section className={styles.privacyWrapper}>
+      <h1>Privacy Policy</h1>
```

Comment:
see below

Reply:
https://seonomad.net/article/mify-o-zagolovkah
This is not a critical error. That h1 tags are under the section tag.